tests/e2e/specs/user-input-questions.test.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * User Input Settings tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createURL, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	activatePlugins,
+	deactivateUtilityPlugins,
+	resetSiteKit,
+	setupSiteKit,
+	useRequestInterception,
+	enableFeature,
+} from '../utils';
+
+describe( 'User Input Settings', () => {
+	async function fillInInputSettings() {
+		await page.waitForSelector( '.googlesitekit-user-input__question' );
+
+		await expect( page ).toClick( '#role-owner_with_team' );
+		await expect( page ).toClick( '.googlesitekit-user-input__buttons--next' );
+
+		await expect( page ).toClick( '#postFrequency-monthly' );
+		await expect( page ).toClick( '.googlesitekit-user-input__buttons--next' );
+
+		await expect( page ).toClick( '#goals-publish_blog' );
+		await expect( page ).toClick( '#goals-share_portfolio' );
+		await expect( page ).toClick( '.googlesitekit-user-input__buttons--next' );
+
+		await expect( page ).toClick( '#helpNeeded-retaining_visitors' );
+		await expect( page ).toClick( '#helpNeeded-improving_performance' );
+		await expect( page ).toClick( '#helpNeeded-help_better_rank' );
+		await expect( page ).toClick( '.googlesitekit-user-input__buttons--next' );
+
+		await expect( page ).toFill( '#searchTerms-keywords', 'One,Two,Three,' );
+		await expect( page ).toClick( '.googlesitekit-user-input__buttons--next' );
+
+		await Promise.all( [
+			expect( page ).toClick( '.googlesitekit-user-input__buttons--next' ),
+			page.waitForNavigation(),
+		] );
+	}
+
+	async function expectSuccessNotification() {
+		await page.waitForSelector( '#user-input-success' );
+		await expect( page ).toMatchElement( '#user-input-success' );
```

Comment:
This is redundant here as `waitForSelector` will fail the test if it times out (default 30s).

We shouldn't need to wait that long so I think it would be better to use just the `toMatchElement` here, but using the `timeout` option to use a more reasonable limit in the event of a legitimate failure (maybe 5s?).

Reply:
This is needed because we should wait till the page is redirected. If I remove it, tests fail.

Reply:
What we can remove is the expect clause: `await expect( page ).toMatchElement( '#user-input-success' );`. Updated.